### ISSUE TITLE
Integration test: unsupported kubernetes version "v1.22.2"

### ISSUE THREAD

**Issue by ialidzhikov** [gardener/gardener-extension-runtime-gvisor#27]
/area testing
/kind bug
/priority 3

**What happened**:
With `runtime-gvisor@v0.2.0` the integration test fails against K8s v1.22 Shoot with reason:
```
• Failure in Spec Setup (BeforeEach) [300.835 seconds]
gVisor tests
/src/test/integration/container-runtime/container_runtime.go:40
  [BETA] [SERIAL] [SHOOT] should add, remove and upgrade worker pool with gVisor [BeforeEach]
  /src/vendor/github.com/gardener/gardener/test/framework/gingko_utils.go:26

  Unexpected error:
      <*retry.Error | 0xc00084c540>: {
          ctxError: <context.deadlineExceededError>{},
          err: <*fmt.wrapError | 0xc00084c400>{
              msg: "could not construct Shoot client: error discovering kubernetes version: unsupported kubernetes version \"v1.22.2\"",
              err: <*fmt.wrapError | 0xc00084c3e0>{
                  msg: "error discovering kubernetes version: unsupported kubernetes version \"v1.22.2\"",
                  err: <*errors.errorString | 0xc000639040>{
                      s: "unsupported kubernetes version \"v1.22.2\"",
                  },
              },
          },
      }
      retry failed with context deadline exceeded, last error: could not construct Shoot client: error discovering kubernetes version: unsupported kubernetes version "v1.22.2"
  occurred

  /src/vendor/github.com/gardener/gardener/test/framework/shootframework.go:119
```

It seems that this issue is fixed on the master branch thanks to #26. So it would be nice a have a new release of runtime-gvisor where the test does not complain about the Shoot K8s version.
In longterm, if possible, the test can be adapted to do not rely on the supported K8s versions list in gardener. Similar hidden dependency was present for the networking extensions and it was resolved for example with gardener/gardener-extension-networking-calico#111.

**What you expected to happen**:
No similar error when running the integration test.

**How to reproduce it (as minimally and precisely as possible)**:
See above.

**Environment**:

- Gardener version (if relevant):
- Extension version: v0.2.0
- Kubernetes version (use `kubectl version`):
- Cloud provider or hardware configuration:
- Others:

**Comment by vpnachev** [comment]
This error seems to originate from https://github.com/gardener/gardener/blob/744011e6bc35b3e17a01f41e4cfa778575aba2a5/test/framework/shootframework.go#L216. Should we move this issue to g/g?

**Comment by vpnachev** [comment]
Now this test is failing for k8s v1.24.3. 
It should be fixed by https://github.com/gardener/gardener-extension-runtime-gvisor/pull/55, but there is no release with this change.

@gardener/gardener-maintainers can you please take care and release a new version of this extension?

**Comment by vpnachev** [comment]
Now when [v0.6.0](https://github.com/gardener/gardener-extension-runtime-gvisor/releases/tag/v0.6.0) is released, should we close this issue?

**Comment by MrBatschner** [comment]
Fixed with dependabot introduced in PR #86 .

/close
